calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# TigeraStatus
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The operator get its configuration from the API resources and communicates through the Kubernetes API. The operator is deployed as a Deployment in the `tigera-operator` namespace, and records status in the `tigerastatus` resource. Depending on how your cluster is configured, you can get `tigerastatus` on these resources:
+
+- apiserver
+- applicationlayer
+- authentication
+- calico
+- calico-windows
+- compliance
+- egressgateway
+- intrusion-detection
+- log-collector
+- log-storage
+- manager
+- management-cluster-connection
+- policy-recommendation
+- secrets
+
+## Get TigeraStatus
+
+To get tigerastatus for a resource, run the following command:
+
+   `kubectl get tigerastatus`
+
+For detailed output (including messages and further details on any non-functioning components), run either of the following commands:
+
+   `kubectl describe tigerastatus X`
+
+   `kubectl get tigerastatus X -o yaml`
+
+## Log storage
+
+Log storage provides persistent storage for {{prodname}} Elasticsearch logs (flow, dns, l7, bpg, audit, etc.), and compliance reports.
+
+To check log storage status, run the following command:
+
+   `kubectl get logstorage.operator.tigera.io tigera-secure`
+
+The following table lists the type of information displayed in TigerStatus for log storage.
+
+| TigeraStatus name           | Displays status of...                                        | Standalone             | Managed cluster             | Management cluster       |
+| --------------------------- | ------------------------------------------------------------ | ---------------------- | --------------------------- | ------------------------ |
+| log-storage                 | LogStorage subsystem.                                        | <center>  ✔</center>   |                             | <center>✔</center>       |
+| log-storage-elastic         | Elasticsearch cluster and Kibana.                            | <center>    ✔</center> |                             | <center>      ✔</center> |
+| log-storage-kubecontrollers | Elasticsearch gateway and kube-controllers.                  | <center>✔</center>     |                             | <center>✔</center>       |
+| log-storage access          | {{prodname}} log storage API service.                        | <center>✔</center>     | <center>         ✔</center> |                          |
```

Comment:
missing a hyphen on this one

Reply:
log-storage-access should be standalone and management, not managed.

Reply:
Fixed.